src/impedance_correction.jl

```diff
@@ -0,0 +1,41 @@
+"""
+Attribute that contains information regarding the Transformer Impedance Correction Table.
+
+# Arguments
+- `ict_row::Int64`: Row number of the impedance correction table to linked it with a specific Transformer component.
+- `internal::InfrastructureSystemsInternal`: power system internal reference, do not modify
+"""
+struct ImpedanceCorrectionTable <: SupplementalAttribute
```

Comment:
I think this represents just one row of the impedance correction table, not the entire table, right? Maybe the name should reflect that

Reply:
Yes, this should be `ImpedanceCorrectionData`